tests/posix/common/src/nanosleep.c

```diff
@@ -0,0 +1,221 @@
+/*
+ * Copyright (c) 2018 Friedt Professional Engineering Services, Inc
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <ztest.h>
+#include <errno.h>
+#include <posix/time.h>
+
+#define NS_PER_SEC 1000000000ULL
```

Comment:
We should use here `NSEC_PER_SEC`

Reply:
Sorry, I thought I had fixed that already, thanks for pointing it out.